ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -323,6 +349,14 @@ func convertToContainerDef(context *project.Context, inputCfg *config.ServiceCon
 		outputContDef.LinuxParameters.Capabilities.SetDrop(aws.StringSlice(inputCfg.CapDrop))
 	}
 
+	if ecsContainerDef != nil {
+	// fmt.Printf("CONVERTING CONTAINER_DEF: %+v, TO ECS CONTAINER_DEF: %+v\n", outputContDef, ecsContainerDef)
```

Comment:
ha ha you should probably remove this commented out code here and below :P

Reply:
oops. 😳